### PR TITLE
Hotfix/single tab bug

### PR DESCRIPTION
**Pull request by levkz**
Using tabs feature with one tab was breaking it
### Input:
image

### Output:
image

### CODE REVIEW

File: commons/_shared/bs-lib-components/ui/src/components/layout/BsLayoutDefault.vue [hunk 1]
```diff
@@ -197,6 +190,12 @@ export default defineComponent({
             };
         },
     },
+    watch: {
+        "tabs.length"(newVal: number) {
+            this.defaultTabUsed = (newVal === 0) || ((newVal === 1) && (this.tabs[0].name === this.defaultLayoutTabName));
+            console.log("this.defaultTabUsed: ", this.defaultTabUsed);
```
Comment (HenriChabert) [inline commons/_shared/bs-lib-components/ui/src/components/layout/BsLayoutDefault.vue:196]: Debug msg

File: commons/_shared/bs-lib-components/ui/src/components/layout/BsTab.vue [hunk 2]
```diff
@@ -174,7 +180,11 @@ export default defineComponent({
     },
     emits: ['mounted:q-page'],
     mounted() {
+        console.log("mounted a tab: ", this.tab);
```
Comment (HenriChabert) [inline commons/_shared/bs-lib-components/ui/src/components/layout/BsTab.vue:183]: Debug msg

**Review by HenriChabert — CHANGES_REQUESTED** [review]
Just two debug messages, otherwise, tested and everything works fine!

File: commons/_shared/bs-lib-components/ui/src/components/layout/BsLayoutDefault.vue (hunk 1 above)
Comment (levkz) [reply commons/_shared/bs-lib-components/ui/src/components/layout/BsLayoutDefault.vue:196]: changed in 9602ae0

**Review by HenriChabert — APPROVED** [review]

**Pull request merged**